app.config.js

```diff
@@ -11,8 +11,7 @@ module.exports = {
     process.env.GATSBY_METADATACACHE_URI || 'https://aquarius.delta-dao.com',
 
   complianceUri:
-    process.env.GATSBY_COMPLIANCE_URI ||
-    'https://cs.gx.test.sphereon.com/api/2210vp',
+    process.env.GATSBY_COMPLIANCE_URI || 'https://cs.gx.test.sphereon.com/api',
```

Comment:
I'm missing something here.. what is the role of fma compliance here?

Reply:
I don't think this question is related to the changes in the code. So, let's move on with the review and check with Niels later?